css/index.css

```
@@ -0,0 +1,54 @@
+body {
+    background-color: #efdda8;
+    color: #333;
+    font-family: Arial, sans-serif;
+    margin:20px;
+}
+
+h1, h2, h3 {
+    font-size: 24px;
+    font-weight:bold;
+    color: #1c1b1b;
+}
+h1 {
+    text-align: center;
+    font-size: 35px;
+}
+.exprience {
+    margin-bottom: 10px;
+    background-color: #f1f1f1;
+    padding:10px;
+    border-radius: 4px;
+}
+#skills {
```

```suggestion
.exprience, #skills {
```
If you want the same style, feel free to combine selectors.